src/main/java/com/github/dockerjava/core/DockerClientImpl.java

```diff
@@ -77,16 +86,8 @@
 import com.github.dockerjava.core.command.VersionCmdImpl;
 import com.github.dockerjava.core.command.WaitContainerCmdImpl;
 
-import java.io.Closeable;
```

Comment:
I'm using Intellij, it's re-arranged automatically like that by default. Please feel free to re-organize.